federation/apis/swagger-spec/api.json

```diff
@@ -3,6 +3,10 @@
   "apiVersion": "",
   "basePath": "https://10.10.10.10:6443",
   "resourcePath": "/api",
+  "info": {
```

Comment:
These got added as a result of https://github.com/kubernetes/kubernetes/pull/30914.

These fields are part of the spec, go-restful might have recently added support for it.